[2018-10-05 17:33:26] api-log.ERROR: Exception occurred on create order:Field store_id is mandatory.  [] []
[2018-10-05 17:33:26] api-log.ERROR: Exception occurred on create order:Field store_id must be numeric.  [] []
[2018-10-05 17:34:18] api-log.ERROR: Exception occurred on create order:Field id is mandatory.  [] []
[2018-10-05 17:34:18] api-log.ERROR: Exception occurred on create order:Field id must be numeric.  [] []
[2018-10-05 17:39:24] api-log.ERROR: Exception occurred on create order:Lines number must be sequential, starting at 1 [] []
[2018-10-05 17:39:24] api-log.ERROR: Exception occurred on create order service:Error on validate Order Entity [] []
[2018-10-05 17:42:55] api-log.ERROR: Exception occurred on create order:field order is mandatory [] []
[2018-10-05 17:44:30] api-log.ERROR: Exception occurred on create order:Invalid Json [] []
[2018-10-05 17:54:27] api-log.ERROR: Exception occurred on create order:Lines number must be sequential, starting at 1 [] []
[2018-10-05 17:54:27] api-log.ERROR: Exception occurred on create order service:Error on validate Order Entity [] []
[2018-10-05 17:54:37] api-log.ERROR: Exception occurred on create order:Lines number must be sequential, starting at 1 [] []
[2018-10-05 17:54:37] api-log.ERROR: Exception occurred on create order service:Error on validate Order Entity [] []
[2018-10-05 17:54:47] api-log.ERROR: Exception occurred on create order:Lines number must be sequential, starting at 1 [] []
[2018-10-05 17:54:47] api-log.ERROR: Exception occurred on create order service:Error on validate Order Entity [] []
[2018-10-05 17:55:09] api-log.ERROR: Exception occurred on create order service:SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '2-1' for key 'PRIMARY' [] []
[2018-10-05 17:56:37] api-log.ERROR: Exception occurred on create order service:SQLSTATE[42S22]: Column not found: 1054 Unknown column 'store' in 'where clause' [] []
[2018-10-05 17:58:24] api-log.ERROR: Exception occurred on create order service:SQLSTATE[42S22]: Column not found: 1054 Unknown column 'store' in 'where clause' [] []
[2018-10-05 17:59:01] api-log.ERROR: Exception occurred on create order:Order with id: [2] it already exists in store with id: [1] [] []
[2018-10-05 17:59:01] api-log.ERROR: Exception occurred on create order service:Error on validate Order Entity [] []
[2018-10-05 17:59:18] api-log.ERROR: Exception occurred on create order:Order with id: [2] it already exists in store with id: [2] [] []
[2018-10-05 17:59:18] api-log.ERROR: Exception occurred on create order service:Error on validate Order Entity [] []
[2018-10-05 18:04:15] api-log.ERROR: Exception occurred on create order service:SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`oms`.`order`, CONSTRAINT `fk_store_id` FOREIGN KEY (`store_id`) REFERENCES `store` (`id`) ON UPDATE CASCADE) [] []
[2018-10-05 18:13:07] api-log.ERROR: Exception occurred on create order service:SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`oms`.`order`, CONSTRAINT `fk_store_id` FOREIGN KEY (`store_id`) REFERENCES `store` (`id`) ON UPDATE CASCADE) [] []
[2018-10-05 18:13:28] api-log.ERROR: Exception occurred on create order:Store with id: [4] does not exists. [] []
[2018-10-05 18:13:28] api-log.ERROR: Exception occurred on create order service:Error on validate Order Entity [] []
[2018-10-05 18:20:50] api-log.ERROR: Exception occurred on create order:Invalid Json [] []
[2018-10-05 18:20:57] api-log.ERROR: Exception occurred on create order:Store with id: [4] does not exist. [] []
[2018-10-05 18:20:57] api-log.ERROR: Exception occurred on create order service:Error on validate Order Entity [] []
[2018-10-05 19:23:39] api-log.ERROR: Exception occurred on create order:Store with id: [4] does not exist. [] []
[2018-10-05 19:23:39] api-log.ERROR: Exception occurred on create order service:Error on validate Store [] []
[2018-10-05 19:25:59] api-log.ERROR: Exception occurred on create order:Store with id: [4] does not exist. [] []
[2018-10-05 19:25:59] api-log.ERROR: Exception occurred on create order service:Error on validate Store [] []
[2018-10-05 19:54:33] api-log.ERROR: Exception occurred on create order:Store with id: [4] does not exist. [] []
[2018-10-05 19:54:33] api-log.ERROR: Exception occurred on create order service:Error on validate Store [] []
[2018-10-05 20:33:45] api-log.ERROR: Exception occurred on create order:Field store_id is mandatory.  [] []
[2018-10-05 20:33:45] api-log.ERROR: Exception occurred on create order:Field store_id must be numeric.  [] []
[2018-10-05 20:47:06] api-log.ERROR: Exception occurred on create order:Store with id: [4] does not exist. [] []
[2018-10-05 20:47:06] api-log.ERROR: Exception occurred on create order service:Error on validate Store [] []
[2018-10-05 20:58:04] api-log.ERROR: Exception occurred on create order:Invalid Json [] []
